addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/messages/RFXComChimeMessage.java

```diff
@@ -0,0 +1,235 @@
+
+    private static final List<RFXComValueSelector> SUPPORTED_OUTPUT_VALUE_SELECTORS = Collections
+            .singletonList(RFXComValueSelector.CHIME_SOUND);
+
+    public SubType subType = SubType.UNKNOWN;
+    public int sensorId = 0;
+    public int chimeSound = 0;
+    public byte signalLevel = 0;
+
+    public RFXComChimeMessage() {
+        packetType = PacketType.CHIME;
+    }
+
+    public RFXComChimeMessage(byte[] data) throws RFXComException {
+        encodeMessage(data);
+    }
+
+    @Override
+    public String toString() {
+        String str = "";
+
+        str += super.toString();
+        str += ", Sub type = " + subType;
+        str += ", Device Id = " + getDeviceId();
+        str += ", Chime Sound = " + chimeSound;
+        str += ", Signal level = " + signalLevel;
+
+        return str;
+    }
+
+    @Override
+    public void encodeMessage(byte[] data) throws RFXComException {
+
+        super.encodeMessage(data);
+
+        subType = SubType.fromByte(super.subType);
+
+        switch (subType) {
+            case BYRONSX:
+                sensorId = (data[4] & 0xFF) << 8 | (data[5] & 0xFF);
+                chimeSound = data[6];
+                break;
+            case BYRONMP001:
+            case SELECTPLUS:
+            case SELECTPLUS3:
+            case ENVIVO:
+                sensorId = (data[4] & 0xFF) << 16 | (data[5] & 0xFF) << 8 | (data[6] & 0xFF);
+                chimeSound = 1;
+                break;
+        }
+
+        signalLevel = (byte) ((data[7] & 0xF0) >> 4);
+    }
+
+    @Override
+    public byte[] decodeMessage() {
+        byte[] data = new byte[8];
+
+        data[0] = 0x07;
+        data[1] = packetType.toByte();
+        data[2] = subType.toByte();
+        data[3] = seqNbr;
+
+        switch (subType) {
+            case BYRONSX:
+                data[4] = (byte) ((sensorId & 0xFF00) >> 8);
+                data[5] = (byte) (sensorId & 0x00FF);
+                data[6] = (byte) chimeSound;
+                break;
+            case BYRONMP001:
+            case SELECTPLUS:
+            case SELECTPLUS3:
+            case ENVIVO:
+                data[4] = (byte) ((sensorId & 0xFF0000) >> 16);
+                data[5] = (byte) ((sensorId & 0x00FF00) >> 8);
+                data[6] = (byte) ((sensorId & 0x0000FF));
+                break;
+        }
+
+        data[7] = (byte) ((signalLevel & 0x0F) << 4);
+
+        return data;
+    }
+
+    @Override
+    public String getDeviceId() {
+        return String.valueOf(sensorId);
+    }
+
+    @Override
+    public State convertToState(RFXComValueSelector valueSelector) throws RFXComException {
+
+        State state = UnDefType.UNDEF;
+
+        if (valueSelector.getItemClass() == NumberItem.class) {
+
+            if (valueSelector == RFXComValueSelector.SIGNAL_LEVEL) {
+
+                state = new DecimalType(signalLevel);
+
+            } else if (valueSelector == RFXComValueSelector.CHIME_SOUND) {

```

Comment:
Do we want to limit returning this only to 'Byron SX'. Or should we just keep returning 0  here in that case? We could also limit the supported output channels returned for the other types?